### DISCUSSION TITLE
计算任务执行失败

### DISCUSSION THREAD

**Discussion by soonxf** [1Panel-dev/1Panel#6306] [Q&A]
### 联系方式

soonxf@digntalk.com

### 1Panel 版本

v1.10.14-lts

### 问题描述

* 发行版本 centos 7.9
* 内核版本 6.9.7-1.el7.elrepo.x86_64
* 系统类型 x86_64

定时执行的计算任务多次都执行失败, shutdown poweroff init 0 多种命令都试过,上个版本 init 0 成功过,昨天更新最新版本 init 0 也不行了,不知道是更新的版本还是之前的执行成功也是概率问题

![image](https://github.com/user-attachments/assets/bcf77630-140c-40bf-aef1-80e805ebe83d)
![image](https://github.com/user-attachments/assets/f003b4cb-83ac-4281-b09d-75a41ca5d395)
![image](https://github.com/user-attachments/assets/84016a9f-955f-4a42-8eb8-67a7df15b84e)
![image](https://github.com/user-attachments/assets/1994927c-0180-47d6-a0c3-a66395f95434)

### 重现步骤

创建计划任务执行关机命令

**Comment by wanghe-fit2cloud** [comment]
在本地的 CentOS 7.9 和 Ubuntu 22.04 环境中均未复现该问题。
> 建议你先检查一下 1Panel 系统日志（日志审计页面）和服务器的系统日志，看看是否有相关的错误提示。
